Foreground.skin.php

```diff
@@ -199,11 +199,11 @@ public function execute() {
 		<div class="row">
 				<div id="p-cactions" class="large-12 columns">
 					<?php if ($wgUser->isLoggedIn() || $wgForegroundFeatures['showActionsForAnon']): ?>
-						<a href="#" data-dropdown="drop1" class="button dropdown small secondary radius"><i class="fa fa-cog"><span class="show-for-medium-up">&nbsp;<?php echo wfMessage( 'actions' )->text() ?></span></i></a>
+						<a href="#" class="button dropdown small secondary radius"><i class="fa fa-cog"><span class="show-for-medium-up">&nbsp;<?php echo wfMessage( 'actions' )->text() ?></span></i></a>
```

Comment:
Ive found a way by doing hover_timeout:100; instead. Also the action button seems to be fixed in foundation5 since followed your suggestion and it fixed the problem. And I also did some minor changes and seems js is working much better more stable and improved performance since it is loading less modules.